app/controllers/bookmarks_controller.rb

```diff
@@ -46,10 +46,10 @@ def destroy
     end
   end
 
-end
+	private
 
-private
+	def bookmark_params
+		params.require(:bookmark).permit(:url, :title)
+	end
```

Comment:
Good work bringing this in to the controller. Consider tweaking the indentation for this file, see how it is on GitHub: https://github.com/tnum/blocmarks/blob/fixing-errors/app/controllers/bookmarks_controller.rb